prow/config/config.go

```diff
@@ -97,6 +97,10 @@ type Plank struct {
 	// will be passed a kube.ProwJob and can provide an optional blurb below
 	// the test failures comment.
 	ReportTemplate *template.Template `json:"-"`
+
+	// MaxConcurrency is the maximum number of tests running concurrently that
+	// will be allowed by plank. 0 implies no limit.
+	MaxConcurrency int `json:"max_concurrency,omitempty"`
```

Comment:
Still feel like we should embed a common struct for this stuff, not clear what we gain with the copy paste

Reply:
How would we use separate limits for Jenkins and k8s in that case?

Reply:
I mean Go embedding

Reply:
Yeah, that's an option too but I would prefer to tackle it in a follow-up. Included in https://github.com/kubernetes/test-infra/issues/4744.